src/backend/tests/models/charge_types/test_schedule_1_p_c_s.py

```diff
@@ -68,3 +68,28 @@ def test_pcs_475992():
     assert isinstance(pcs_charge, Schedule1PCS)
     assert pcs_charge.expungement_result.type_eligibility.status is EligibilityStatus.ELIGIBLE
     assert pcs_charge.expungement_result.type_eligibility.reason == "Eligible under 137.225(5)(c)"
+
+
+def test_pcs_dismissed_violation():
+    charge_dict = ChargeFactory.default_dict()
+    charge_dict["name"] = "Poss Controlled Sub 2"
+    charge_dict["statute"] = "4759924B"
+    charge_dict["disposition"] = Dispositions.DISMISSED
+    for level in ("Felony Class viOlation C", "Felony Violation", "violation Felony", "some other Violation"):
```

Comment:
Ah shoot. Just saw this too. There's no such thing as a Felony Violation, so these test cases seem weird. If we did handle them, we'd probably report an OECI data error, but I don't think we need to bother. So I would probably just take them out.

Reply:
But "Class C Violation" seems like a plausible string to check for

Reply:
updated with sensible test cases